### ISSUE TITLE
OAI: theses absent in oai_dc responses

### ISSUE THREAD

**Issue by anayram** [ualbertalib/jupiter#1902]
**Describe the bug**
ERA theses are not included when metadataPrefix is set to oai_dc

**To Reproduce**
Examples of requests to reproduce this:
- Thesis record in oai_etdms format: https://era-app-stg-1.library.ualberta.ca/oai?verb=GetRecord&metadataPrefix=oai_etdms&identifier=oai:era.library.ualberta.ca:9fdae4cf-ba36-4a69-a78b-dab539ad2b05
- Request for same record in oai_dc format: https://era-app-stg-1.library.ualberta.ca/oai?verb=GetRecord&metadataPrefix=oai_dc&identifier=oai:era.library.ualberta.ca:9fdae4cf-ba36-4a69-a78b-dab539ad2b05
- No theses included in complete harvest. Link to first page of results: https://era-app-stg-1.library.ualberta.ca/oai?verb=ListRecords&metadataPrefix=oai_dc

**Expected behavior**
All ERA items to be included in oai_dc responses regardless of item type.

**Comment by mbarnett** [comment]
Good catch!

Am I right in assuming that LAC is only harvesting ETD-MS? If so I'm thinking we can go ahead with this right now and then circle back and add an OAI_DC renderer for the Theses once we're out to prod, rather than delay the migration further to round out our general OAI support (there's a couple of other areas where we're out of spec right now around flagging request errors that we need to clean up but which aren't an immediate priority because they don't impact LAC, so we can mop them up after launch).

**Comment by anayram** [comment]
@mbarnett yes, that is right, as far as I know LAC is only harvesting ETD-MS according to latest communications and their documentation at https://www.bac-lac.gc.ca/eng/services/theses/Pages/universities.aspx (although they change their requirements very frequently and often there is a delay in documentation updates). So afaik oai_dc can come later. Looping in @sfarnel and @seanluyk in case there is anything to note here.

**Comment by sfarnel** [comment]
Thanks @anayram @mbarnett This matches my understanding as well, and seems like a solid plan.

**Comment by anayram** [comment]
@pgwillia flagging this issue as well. The problem happens with GetRecord but it could happen with other requests.

To reproduce, see this thesis in OAI `GetRecord` response with `metadataPrefix` set to `oai_etdms`:
https://era.library.ualberta.ca/oai?verb=GetRecord&metadataPrefix=oai_etdms&identifier=oai:era.library.ualberta.ca:269f3d9e-eecb-4902-84eb-16cd70ff7347

See same oai request but with `metadataPrefix` set to `oai_dc`:
https://era.library.ualberta.ca/oai?verb=GetRecord&metadataPrefix=oai_dc&identifier=oai:era.library.ualberta.ca:269f3d9e-eecb-4902-84eb-16cd70ff7347

However, when using `ListRecords` verb with `set` specified, theses are returned just fine and in any format. For instance, with this set's id:
```xml
<set>
**<setSpec>db9a4e71-f809-4385-a274-048f28eb6814:f42f3da6-00c3-4581-b785-63725c33c7ce</setSpec>**
<setName>Graduate Studies and Research, Faculty of / Theses and Dissertations</setName>
<setDescription>
<oai_dc:dc xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/oai_dc/ http://www.openarchives.org/OAI/2.0/oai_dc.xsd">
<dc:description>This collection contains theses and dissertations of graduate students of the University of Alberta. The collection contains a very large number of theses electronically available that were granted from 1947's to 2009, 90% of theses granted from 2009-2014, and 100% of theses granted from April 2014 to the present (as long as the theses are not under temporary embargo by agreement with the Faculty of Graduate Studies and Research). IMPORTANT NOTE: To conduct a comprehensive search of all UofA theses granted and in University of Alberta Libraries collections, search the library catalogue at www.library.ualberta.ca - you may search by Author, Title, Keyword, or search by Department. To retrieve all theses and dissertations associated with a specific department from the library catalogue, choose 'Advanced' and keyword search "university of alberta dept of english" OR "university of alberta department of english" (for example). Past graduates who wish to have their thesis or dissertation added to this collection can contact the ERA Mediated HelpDesk at erahelp@ualberta.ca. </dc:description>
</oai_dc:dc>
</setDescription>
</set>
```
Theses are returned both in `oai_dc`:
https://era.library.ualberta.ca/oai?verb=ListRecords&metadataPrefix=oai_dc&set=db9a4e71-f809-4385-a274-048f28eb6814:f42f3da6-00c3-4581-b785-63725c33c7ce
And in `oai_etdms`:
https://era.library.ualberta.ca/oai?verb=ListRecords&metadataPrefix=oai_etdms&set=db9a4e71-f809-4385-a274-048f28eb6814:f42f3da6-00c3-4581-b785-63725c33c7ce

This fix is not needed for LAC harvest.
